Add Alibaba Cloud Nacos binding Doc (#1401)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md

---
type: docs
title: "Alibaba Cloud Nacos binding spec"
linkTitle: "Alibaba Cloud Nacos"
description: "Detailed documentation on the Alibaba Cloud Nacos binding component"

> aliases:
> - "/operations/components/setup-bindings/supported-bindings/alicloud-nacos/"

Reviewer: ```suggestion
```

You can delete this as it's the first time this page is getting created and there wasn't a prior location.

| updateCacheWhenEmpty                | N        | Input/Output |update cache when get empty service instance from server | `"false"`
| username                | N        | Input/Output |the password for Nacos auth | `"username"`
| password                | N        | Input/Output |the password for Nacos auth | `"password"`
| logDir                | N        | Input/Output |the directory for log, default is current path | `"/tmp/nacos/log"`
| rotateTime                | N        | Input/Output |the rotate time for log, eg: 30m, 1h, 24h, default is 24h| `"1h"`
| maxAge                | N        | Input/Output |the max age of a log file, default value is 3 | `3`
| logLevel                | N        | Input/Output |the level of log, it's must be debug,info,warn,error, default value is info | `"info"`
| watches                | N        | Input |A comma separated string of watch keys | `"key1:group1,key2:group2"`

## Binding support

This component supports both **input and output** binding interfaces.

This component supports **output binding** with the following operations:
- `create`
- `get`

## Create an AliCloud Nacos

> Follow the instructions [here](https://www.alibabacloud.com/help/doc-detail/59963.htm) on setting up AliCloud nacos.
> Follow the instructions [here](https://help.aliyun.com/document_detail/85466.html) on setting up AliCloud nacos.

Reviewer: ```suggestion
Follow [these instructions](https://www.alibabacloud.com/help/doc-detail/59963.htm) to setup AliCloud Nacos.
   - A Chinese (simplified) version can be found [here](https://help.aliyun.com/document_detail/85466.html)
```

Added suggestion for grammer. But curious why these links go to ACM. Shouldn't the Nacos link be https://nacos.io/en-us/docs/quick-start.html?

## Specifying a partition key

> When invoking the nacos binding, its possible to provide an optional partition key by using the `metadata` section in the request body.

Reviewer: ```suggestion
When invoking the Nacos binding, its possible to provide an optional partition key by using the `metadata` section in the request body.
```
